pipelines/push-to-external-registry/push-to-external-registry.yaml

```diff
@@ -59,6 +59,7 @@ spec:
     - name: release-workspace
   tasks:
     - name: verify-access-to-resources
+      retries: 5
```

Comment:
Wouldn't it make more sense to add retries only to places where we know things sometimes fail? I can't imagine a situation where `verify-access-to-resources` would fail and a retry would fix the problem. Have we seen it before?

Same with `collect-data`, `reduce-snapshot`, `apply-mapping`, `collect-registry-token-secret`.

Reply:
I can remove the ones that seem unlikely to fail...but is there any harm in having the retries? It is not like it will take more resources...

Reply:
I guess you can look at it from both sides. One is: Make the code robust, so add retries as a preventive measure even if we haven't seen any issues yet. The other is: Don't add retry unless you've seen issues somewhere.

In general the first approach probably makes more sense. But I just think that for tasks that don't make any api calls (other than kubernetes which we don't expect to fail randomly), we don't really need it. Because these tasks do fail - e.g. if you have wrong mapping in RPA, then apply-mapping will fail. But when they fail, having retry will just add noise in the log.

So yeah, I would remove it from these tasks, because AFAICS, they don't make any api calls.